tools/config/qradar.yml

```diff
@@ -37,6 +37,7 @@ fieldmappings:
  c-uri: URL
  c-uri-extension: URL
  c-useragent: user_agent
```

Comment:
Why not remove the old one? It's confusing now.

Reply:
For the proxy category